README.md

```diff
@@ -25,6 +25,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 | Build | Status |
 | ---------------------- | -------------------- |
 | **Travis CI** | [![Travis Status](https://api.travis-ci.org/eclipse/omr.svg?branch=master)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux x86-64 CMAKE Makefile Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/1)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux x86-64 Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/2)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux AArch64 Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/3)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux ARM Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/4)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux x86-64 LINT Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/5)](https://travis-ci.org/eclipse/omr) |
+| **Travis CI Linux x86-64 CMAKE Ninja Matrix Build** | [![Travis Status](https://travis-matrix-badges.herokuapp.com/repos/eclipse/omr/branches/master/6)](https://travis-ci.org/eclipse/omr) |
```

Comment:
It looks like all these links are pointing to the same location. Are all of them really needed?

Reply:
Each build badge is pointing to a separate one of the 6 matrix build in Travis. If one out of the 6 matrix builds fails it will explicitly show the one that is failing. The Travis build badge we have now is more of a black box and doesn't indicate that only one/some matrix build/platform failed or which one in particular failed. The URLs that the build badges "click to" are all the main eclipse/omr [Travis page](https://travis-ci.org/eclipse/omr) as that would allow one to dig deeper into a failure.

Reply:
I see. In that case, I think the approach suggested by @jeanlego is tidier.